mobile/examples/phi-3/ios/LocalLLM/LocalLLM/README.md

```diff
@@ -0,0 +1,71 @@
+# **Local LLM sample application running Phi3-mini on iOS**
+
+## **Steps**
+
+### **A. Preparation**
+
+1. macOS 14+
+
+2. Xcode 15+
+
+3. iOS SDK 17.x (iPhone 14 or iPhone 15 powered by a A16 or A17)
+
+4. Install Python 3.10+ (Conda is recommended)
+
+5. Install the Python library - python-flatbuffers
+
+6. Install CMake
+
+### **B. Compiling ONNX Runtime for iOS**
+
+```bash
+
+git clone https://github.com/microsoft/onnxruntime.git
+
+cd onnxruntime
+
+./build.sh --build_shared_lib --ios --skip_tests --parallel --build_dir ./build_ios --ios --apple_sysroot iphoneos --osx_arch arm64 --apple_deploy_target 16.6 --cmake_generator Xcode --config Release
+
+```
+
+***Notice***
+
+  1. Before compiling, you must ensure that Xcode is configured correctly and set it on the terminal
+
+```bash
+
+sudo xcode-select -switch /Applications/Xcode.app/Contents/Developer 
+
+```
+
+  2. ONNX Runtime needs to be compiled based on different platforms. For iOS, you can compile based on arm64 / x86_64
```

Comment:
Might be good to explain how the user should pick the platform.

x86_64 if they have an Intel mac to run the simulator.
arm64 for an ARM based mac to run the simulator, 
arm64 to run on device.